corehq/apps/userreports/tests/test_rebuild_migrate.py

```diff
@@ -79,39 +150,64 @@ def test_add_non_nullable_column(self):
         insp = reflection.Inspector.from_engine(self.engine)
         table_name = get_table_name(self.config.domain, self.config.table_id)
         self.assertEqual(
-            len([c for c in insp.get_columns(table_name) if c['name'] == 'new_date']), 0
+            len(
+                [
+                    c
+                    for c in insp.get_columns(table_name)
+                    if c['name'] == 'new_date'
+                ]
+            ),
+            0,
         )
```

Comment:
Yeah, this is where `blue` and I disagree. I'd have gone with something like ...
```suggestion
        new_date_cols = [c for c in insp.get_columns(table_name) if c['name'] == 'new_date']
        self.assertEqual(len(new_date_cols), 0)
```